Description: ignore failing tests
 ignore tests failing due to switch to plasma 5
Author: Jonathan Riddell
Origin: me
Forwarded: nope
Last-Update: 2015-09-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: kde-baseapps-15.08.0/konqueror/src/tests/CMakeLists.txt
===================================================================
--- kde-baseapps-15.08.0.orig/konqueror/src/tests/CMakeLists.txt
+++ kde-baseapps-15.08.0/konqueror/src/tests/CMakeLists.txt
@@ -9,12 +9,6 @@ kde4_add_executable(centralwidget TEST c
 target_link_libraries(centralwidget konq ${KDE4_KDECORE_LIBS} ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY}
                       ${QT_QTTEST_LIBRARY})
 
-########### konqviewmgrtest ###############
-
-kde4_add_unit_test(konqviewmgrtest TEST konqviewmgrtest.cpp)
-target_link_libraries(konqviewmgrtest kdeinit_konqueror ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY}
-                      ${KDE4_KHTML_LIBS} ${QT_QTTEST_LIBRARY})
-
 ########### historymanagertest ###############
 
 kde4_add_unit_test(historymanagertest historymanagertest.cpp)
@@ -25,14 +19,4 @@ target_link_libraries(historymanagertest
 kde4_add_unit_test(undomanagertest undomanagertest.cpp)
 target_link_libraries(undomanagertest kdeinit_konqueror ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY})
 
-########### konqhtmltest ###############
-
-kde4_add_unit_test(konqhtmltest konqhtmltest.cpp)
-target_link_libraries(konqhtmltest kdeinit_konqueror ${KDE4_KHTML_LIBS} ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY})
-
-########### konqviewtest ###############
-
-kde4_add_unit_test(konqviewtest konqviewtest.cpp)
-target_link_libraries(konqviewtest kdeinit_konqueror ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY})
-
 endif (NOT WIN32)
